src/main/java/butler/util/Parser.java

```diff
@@ -0,0 +1,136 @@
+package butler.util;
+
+import butler.ButlerInputException;
+import butler.command.Command;
+import butler.command.ByeCommand;
+import butler.command.DeadlineCommand;
+import butler.command.DeleteCommand;
+import butler.command.EventCommand;
+import butler.command.InvalidCommand;
+import butler.command.ListCommand;
+import butler.command.MarkCommand;
+import butler.command.TodoCommand;
+import butler.command.UnmarkCommand;
+import butler.command.FindCommand;
```

Comment:
Shouldn't butler.command.FindCommand be Positioned after EventCommand and InvalidCommand?